### ISSUE TITLE
49.3 : RFE, Option to show unit variant name next to the chassis name

### ISSUE THREAD

**Issue by Question2** [MegaMek/megamek#3202]
Im pretty sure this was the default behaviour in a previous version of megamek, not sure why this has changed...

Right now it simply shows "Archer", "Warhammer", etc in the hex map without the variant names : 

![image](https://user-images.githubusercontent.com/1191378/137148949-a8851c5e-7e75-4e5e-a0bd-340591468810.png)

Mousing over them doesnt show the variant name either, it just shows "Archer (Player Name)". It would be much more informative if there was an option to display the variant name like "Archer ARC-2R" or something like that.

Basically like what an older version of megamek did : 

https://cloud.githubusercontent.com/assets/1191378/21711536/8068fef4-d443-11e6-9962-745d302bc4e5.jpg

**Comment by Question2** [comment]
Thanks, did not realise it was hidden in the key binds, i was checking the other tabs. Default is ctrl+y.

**Comment by SJuliez** [comment]
Changing the label style doesnt work. Some code got lost.
